docs/blog/20240624-Daeraxa-3kStars.md

```diff
@@ -0,0 +1,136 @@
+
+A new Discord server was created for the new community as well as a GitHub organisation in order to organise for the work to come.
+
+### Salvage operations begin
+
+<img src="./assets/atom-packages.png" height="200" />
+
+One of the biggest problems was that time was not on our side. In under six months we would be losing the original Atom.io website and, most importantly, the package registry. The Atom.io package backend was not an open source project like the rest of the editor but was essential in keeping a list of published packages for Atom and serving them to its inbuilt package manager, APM. This is where [@confused-techie] was particularly instrumental in not only the creation of [a tool](https://github.com/confused-Techie/AtomPackagesArchive) to archive the entire package backend from Atom.io but also the, from scratch, re-implementation of the backend service needed to serve the reborn editor.
+
+This was no mean feat and is too much to go into in this blog post, but, thankfully, this is all detailed in our [first ever blog post](https://pulsar-edit.dev/blog/20221127-confused-Techie-SunsetMisadventureBackend.html) if you really want to understand the full scale of the undertaking that was the archival and re-implementation of the Atom backend to turn it into the Pulsar Package Registry.
+
+Of course, there was the main technical challenge, but this wasn't the only obstacle; we had other things that had to be taken into account, like what to do about packages with [no license](https://pulsar-edit.dev/blog/20230319-confused-Techie-HowLicenseNoneDeletedPackages.html), or custom licenses specifically for Atom.io? We also had to deal with a huge amount of spam from Vietnamese and Thai gambling sites that had been hammering the original backend in its final days with no moderation. To indicate the scale of the issue, 12k packages were archived compared to 405k just before atom.io finally shut down.
+
+### Modernizing the editor
+
+<img src="./assets/pulsar1.png" height="200" />
+
+We had some big tasks ahead of us; changes needed to be made, and work needed to start to get some traction on the project. We desperately wanted to keep the community alive, but with the announcement of the sunset, people were fleeing the sinking ship. We had to get the project up and running to show that Pulsar was a serious continuation of the project and not just a fork on GitHub with a renamed readme.
+
+- **Building from source** - Atom had a rather complicated (and now rather fragile) way of [bootstrapping](https://atom-flight-manual-archive.github.io/hacking-atom/sections/hacking-on-atom-core/#cloning-and-bootstrapping) itself for development. One of the first tasks was to simplify this into a process much more familiar to those using Node.js and Electron, which is far more robust and makes it much easier to get involved with Pulsar development.
```

Comment:
```suggestion
- **Building from source:** Atom had a rather complicated (and now rather fragile) way of [bootstrapping](https://atom-flight-manual-archive.github.io/hacking-atom/sections/hacking-on-atom-core/#cloning-and-bootstrapping) itself for development. One of the first tasks was to simplify this to a process that we think is much more robust and much more familiar to those using Node and Electron. This makes it much easier to get involved with Pulsar development.

```

These suggestions add a line break after each list item because I've split up some of the longer ones into several paragraphs.